general/v0.1/registration.md

```diff
@@ -47,8 +45,8 @@ contain the registered claim `exp`.
 These claims MUST contain the following:
 
 * `iss`: The base URI for API requests to the FASP.
-* `sub`: The OAuth 2.0 client identifier
-* `sec`: The OAuth 2.0 client secret
+* `sub`: The client identifier
+* `sec`: The client secret key
```

Comment:
Is this a standard claim? I don't remember seeing it in the IANA registry of JWT claims

Reply:
No it is not. That is why two sentences before this it states:

> The registration token is a JSON Web Token (JWT) that MUST include the registered claims `iss` and `sub` and the private claim `sec`.